System/Remote/Monitoring/Statsd.hs

```diff
@@ -132,19 +142,19 @@ forkStatsd opts store = do
         "unsupported address: " ++ T.unpack (host opts)
     emptySample = M.empty
 
-loop :: Metrics.Store   -- ^ Metric store
-     -> Metrics.Sample  -- ^ Last sampled metrics
-     -> Socket.Socket   -- ^ Connected socket
-     -> StatsdOptions   -- ^ Options
+loop :: Metrics.Store            -- ^ Metric store
```

Comment:
If you add an `Int` parameter to this function and pass it to and return it from `flushSample` you can do something like

```haskell
Socket.sendAll socket msg `catch` \ (e :: IOException) -> do
    when failCount `mod` 100 == 0 $ do
        T.hPutStrLn stderr $ "ERROR: Couldn't send message: " <>
            T.pack (show e) <> "(happened " <> show failCount <> " times)"
    return $! failCount + 1
```